package.json

```diff
@@ -98,8 +102,11 @@
         "@types/react-collapse": "^5.0.1",
         "@types/react-dom": "^17.0.11",
         "@types/react-router-dom": "^5.3.2",
+        "eslint": "^7.11.0",
```

Comment:
Måtte legge til denne som en eksplisitt dependency for å få til å kjøre det med lint-staged. Det er jo ikke helt heldig, men har matched versjonen med react-scripts og når vi går over til next så er det ikke et problem lengre.